Assets/MixedRealityToolkit/Inspectors/MixedRealityToolkit.Core.Inspectors.asmdef

```diff
@@ -6,7 +6,8 @@
         "Microsoft.MixedReality.Toolkit.Core.Build",
         "Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities.Editor",
         "Microsoft.MixedReality.Toolkit.Core.Utilities.Editor",
-        "Microsoft.MixedReality.Toolkit.Core.Utilities.Async"
+        "Microsoft.MixedReality.Toolkit.Core.Utilities.Async",
+        "Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality"
```

Comment:
I'm noticing that after this change, several .asmdef files are including a dependency on Providers.WindowsMixedReality @davidkline-ms  and @keveleigh I don't know much the remifications of including this dependency, is this something that could bite us down the line?

Reply:
Agreed, I'm also not sure this dependency is a good one.

Reply:
@julenka, there should be zero dependencies from the core to any provider or service. Dependencies should go the other direction.

Reply:
It was due to this change here:
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/246dce4c233aaa6350393a5e2a70efeaa94a6887/Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityControllerMappingProfileInspector.cs#L299

It was previously using the removed enum to identify MixedRealityControllers, which is now the concrete type. It's only used to give a different controller name in the inspector in case the handedness is none, then it's supposed to be a Hololens.
I wasn't able to find where to put that distinction but in my opinion this shouldn't be part of any inspector.
Maybe the Hololens is just a different controller instead, then this wasn't necessary.